tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -215,6 +225,7 @@ func (a *awsProvider) createInstance(imageID, instanceType, sshKey string,
 		MaxCount:           aws.Int64(1),
 		NetworkInterfaces:  []*ec2.InstanceNetworkInterfaceSpecification{networkInterface},
 		IamInstanceProfile: iamProfile,
+		UserData:           aws.String(base64.StdEncoding.EncodeToString([]byte(userDataInput))),
```

Comment:
How do you know that this has been executed? Can you add a test which confirms this?